chia/types/chain_state.py

```diff
@@ -0,0 +1,14 @@
+from __future__ import annotations
+
+import dataclasses
+from typing import Optional
+
+from chia.consensus.block_record import BlockRecord
+from chia.util.ints import uint64
+
+
+@dataclasses.dataclass
+class ChainState:
```

Comment:
i think the name is a little confusing especially with the word "state" in the context of blockchain

Reply:
yeah, I'm not super happy with the name either, it's quite generic.

Reply:
maybe ValidationContext or something like that